scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -1099,7 +1099,9 @@ class ScalametaParser(input: Input)(implicit dialect: Dialect) { parser =>
     def unapply(token: Token): Boolean = {
       token.is[soft.KwInline] &&
       (token.next.is[LeftParen] || token.next.is[LeftBrace] || token.next.is[KwNew] ||
-        token.next.is[Ident] || token.next.is[Literal])
+        token.next.is[Ident] || token.next.is[Literal] || token.next.is[Interpolation.Id] ||
```

Comment:
since `token.next` is actually a somewhat non-trivial operation, should we save in a variable or just use
```
  token.is[soft.KwInline] && (token.next match {
    case _: LeftParen | _: LeftBrace | _: KwNew | ... => true
    case _ => false
  })
```

Reply:
Fixed!